src/megachatapi.h

```diff
@@ -2630,6 +2631,14 @@ class MegaChatApi
      */
     void disableVideoCall(MegaChatHandle chatid, bool videoCall, MegaChatRequestListener *listener = NULL);
 
+    /**
+     * @brief Search all audio and video devices at the system at that moment.
```

Comment:
Provide a more detailed explanation, please. Like that existing list will be cleared, or that this function could be used to refresh the existing list in case you have plugged in new audio/video devices